docs/tutorials/game-engines/platform/creating-a-window.md

```diff
@@ -0,0 +1,288 @@
+continue ahead and see the implementation details and explanations.
+
+???+ "Google Searches & You"
+
+    Have you ever heard the expression "Learning how to Google is a skill itself"?
+    There is some truth behind that. One of the issues with learning a dense technology
+    stack like Win32 and OpenGL is that you need to know how to find the *right* documentation
+    amongst all the unimportant ones. You can find a lot of information about any given
+    process if you just know how to search for it.
+
+    You will find a good portion of the Win32 API through Google by prefixing a question
+    with "Win32" before the question such that you get "win32 create a window tutorial"
+    or "win32 how to adjust window size", and the like. You may find yourself on StackOverflow
+    in some instances. StackOverflow is a great resource, but I would recommend you use it
+    to help you find the proper documentation on MSDN.
+
+    MSDN also includes its own search engine powered by Bing (take that as you will). While
+    using this search, you may need to adjust the filter criteria to include "All Documentation"
+    in order to find what you're looking for. As you become more familiar with the Win32 API,
+    you'll begin memorizing common function names but not the parameter layout. In this
+    case, using MSDN's search will be useful to skip straight to the thing you're trying to
+    look up.
+
+Hopefully you've managed to create a window on your own. If not, that's okay too. Microsoft
+designs their documentation to fit all use cases and provides ample features for those cases
+that we aren't necessarily concerned about. It's easy to get lost in the noise, so I will
+walk you through the process.
+
+### Window Classes
+
+The first step to creating a window is to create a "Window Class". A window class defines some
+of the core properties of window. This registration process is straight forward and it lets the
+operating system know what to do when we say "create this window".
+
+[MSDN: Window Class](https://learn.microsoft.com/en-us/windows/win32/api/winuser/ns-winuser-wndclassa)
+
+Take a moment to familiarize yourself with the various struct members using the documentation. Some of them
+provide default behaviors for the window, such as the background color, cursor type,
+and even how the OS should treat the window when you're doing hardware acceleration.
+(This will become important later, but not right now.)
+
+```C++
+
+wWinMain(...)
+{
+
+    // Define the window class name. We will need this for the CreateWindow routine as well.
+    const char* windowClassName = "wcGameEngine";
+
+    // Create a WNDCLASSA instance and define some essential information.
+    WNDCLASSA windowClass = {};
+    windowClass.lpfnWndProc   = WindowProcedure;
+    windowClass.hInstance     = hInstance;
+    windowClass.lpszClassName = windowClassName;
+
+    return 0;
+}
+
+```
+???+ "IMPORTANT: Regarding "A" and "W" in Win32 Function & Struct Names"
```

Comment:
Note that there are different types of admonitions provided by Material for MkDocs https://squidfunk.github.io/mkdocs-material/reference/admonitions/

If you'd like to make this one stand out more, try using the 'warning' template:
```
???+ warning "fizz"
    buzz
```